misc/deploy.sh

```diff
@@ -8,9 +8,9 @@ branch=""
 needsReboot=false
 calFile=""
 
-cd ..
-dirPath="$(pwd)"
-dirName="$(basename `pwd`)"
+export dirPath="$(catkin locate)"
```

Comment:
Does this need to be exported to the environment?
```suggestion
dirPath="$(catkin locate)"
```